modules/openglqt/CMakeLists.txt

```diff
@@ -2,9 +2,12 @@
 # Inviwo OpenGLQt Module
 ivw_module(OpenGLQt)
 
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Svg OpenGL OpenGLWidgets REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Svg OpenGL OpenGLWidgets REQUIRED)
-
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Svg OpenGL REQUIRED)
+if(QT_VERSION_MAJOR EQUAL 6)
+    find_package(Qt6 COMPONENTS Svg OpenGL OpenGLWidgets REQUIRED)
+else()
+    find_package(Qt5 COMPONENTS Svg OpenGL REQUIRED)
```

Comment:
Is this not the same as find_package(QT NAMES Qt6 Qt5 COMPONENTS Svg OpenGL REQUIRED)?

Reply:
for some reason the find package needs to be repeated with the real name, was some note in the qt docs, some variables are not set with the right name otherwise... 
the first one is mostly used to figure out witch qt that is installed...

Reply:
Ok, would be good if a link to it could be added and state that it is a workaround